### ISSUE TITLE
[Governance] Bring together governance docs from cool projects

### ISSUE THREAD

**Issue by RichardLitt** [protontypes/open-sustainable-technology#932]
We need to figure out how to have a good governance plan! 

So, I'm asking: can we each come up with three projects whose (hopefully) minimal governance we would like to look at together to see what they use, so we can copy it? 

I'll start: 

- [PythonNZ](https://python.nz/). They have really excellent financial transparency.
- [Gnome Foundation](https://foundation.gnome.org/). I may be biased here, as I am the Interim Executive Director. 
- [UnifiedJS](https://unifiedjs.com/). @wooorm has been doing great work here setting up a way to not be a BDFL, and I think we should look at it closely.

**Comment by RichardLitt** [comment]
> Here are some good sources that I will read over the next few days to decide what kind of governance plan I favour:

I would worry less about what you favor, and think a bit more about what works best for the entire team, and to see what they think. I do not think there is another way to do this. Top-down approaches to instituting governance don't get buy-in the same way.

**Comment by Ly0n** [comment]
Here are some other good resources that might be helpful:  
https://sustainoss.org/governance-readiness/
https://github.com/jlcanovas/gh-best-practices-template

**Comment by EndlessRecess** [comment]
I have no stake in this but Do-ocracy and "Make process the backup plan, not the default" are really cool ideas.

**Comment by Ly0n** [comment]
> I have no stake in this but Do-ocracy and "Make process the backup plan, not the default" are really cool ideas.

Absolutely! The very first value of the [Agile Mainfesto](https://agilemanifesto.org/) is also very similar:  Individuals and interactions over processes and tools

**Comment by Ly0n** [comment]
This is also a very cool resource: 
https://peps.python.org/pep-8002/

**Comment by Ly0n** [comment]
Here this video also gives good insights about the Python Governance and how it was established: https://www.youtube.com/watch?v=mAC83JVDzL8

**Comment by Ly0n** [comment]
OSI uses this tool for elections, but the project is not open source: https://opavote.com/

**Comment by overflorian** [comment]
See
- talks regarding the creation of a Panoramax foundation https://forum.geocommuns.fr/t/panoramax-la-suite-la-perennisation/2508
- mozilla https://www.mozilla.org

See also (in French):
- open fisca https://openfisca.org/fr/association/statuts
- FPOSM - French Fondation des Professionels OpenStreetMap https://fposm.fr/association
- Framasoft https://framablog.org/2017/09/25/degooglisons-internet-cest-la-fin-du-debut
